dbt_subprojects/dex/models/trades/taiko/platforms/izumi_finance_taiko_base_trades.sql

```diff
@@ -0,0 +1,62 @@
+{{
+    config(
+        schema = 'izumi_finance_taiko',
+        alias = 'base_trades',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['tx_hash', 'evt_index'],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+{% set iziswap_start_date = "2024-05-27" %}  
+
+WITH swaps AS (
+    SELECT
+        contract_address,
+        evt_tx_hash AS tx_hash,
+        evt_index,
+        evt_block_time AS block_time,
+        evt_block_number AS block_number,
+        evt_block_date AS block_date,
+        evt_tx_to AS taker,
+        sellXEarnY,
+        amountX,
+        amountY
+    FROM {{ source('izumi_finance_taiko', 'iziswappool_evt_swap') }}
+    {% if is_incremental() %}
+    WHERE {{ incremental_predicate('evt_block_time') }}
+    {% else %}
+    WHERE evt_block_time >= TIMESTAMP '{{iziswap_start_date}}'
+    {% endif %}
+),
+pools AS (
+    SELECT
+        pool,
+        tokenX,
+        tokenY,
+        fee
+    FROM {{ source('izumi_finance_multichain', 'iziswapfactory_evt_newpool') }}
+)
+SELECT
+    'taiko' AS blockchain,
+    'izumi_finance' AS project,
```

Comment:
```suggestion
    'izumi' AS project,
```

Reply:
only giving one example change here, but we should look to change this across all surface areas in PR